### PR TITLE
Translate Code Style section

### PR DESCRIPTION
**Pull request by alexgalkin**
Another attempt to prepare PR for translation (as mentioned previously SVG will be changed using YAML file later). Please review and comment.

### CODE REVIEW

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 1]
```diff
@@ -269,54 +270,55 @@ If you are writing several "helper" functions and the code that uses them, there
...
+Якщо ви початківець, почніть зі шпаргалки наведеної в початку цього розділу. Потім ви зможете обрати один з існуючих посібників, щоб визначити ті правила, які вам більше підходять.
 
-## Automated Linters
+## Автоматичні засоби перевірки (лінтери)
 
-Linters are tools that can automatically check the style of your code and make improving suggestions.
+Автоматичні засоби перевірки, так звані "лінтери" - це інструменти, що автоматично перевіряють стиль коду та вносять пропозиції щодо його вдосконалення.
 
-The great thing about them is that style-checking can also find some bugs, like typos in variable or function names. Because of this feature, using a linter is recommended even if you don't want to stick to one particular "code style".
+Саме чудове в них є те, зо вони можуть також знайти деякі програмні помилки, наприклад друкарську помилку у назві змінної чи функції. Завдяки цій особливості, рекомендують використовувати лінтер навіть тоді, коли ви не збираєтесь дотримуватись якогось конкретного "стилю коду".
```
Comment (tarasyyyk) [inline 1-js/03-code-quality/02-coding-style/article.md]:
```suggestion
Найкраще в них те, що вони також можуть знайти деякі програмні помилки, наприклад друкарську помилку у назві змінної чи функції. Завдяки цій особливості, рекомендують використовувати лінтер навіть тоді, коли ви не збираєтесь дотримуватись якогось конкретного "стилю коду".
```

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 2]
```diff
@@ -269,54 +270,55 @@ If you are writing several "helper" functions and the code that uses them, there
...
-- [ESLint](http://eslint.org/) -- probably the newest one.
+- [JSLint](http://www.jslint.com/) -- один з перших лінтерів.
+- [JSHint](http://www.jshint.com/) -- має більше налаштувань ніж JSLint.
+- [ESLint](http://eslint.org/) -- напевно, найсучасніший лінтер.
 
-All of them can do the job. The author uses [ESLint](http://eslint.org/).
+Всі вони роблять свою справу. Автор використовує [ESLint](http://eslint.org/).
 
-Most linters are integrated with many popular editors: just enable the plugin in the editor and configure the style.
+Більшість лінтерів інтегровані у популярні редактори: просто ввімкнуть відповідний плагін у редакторі і налаштуйте стиль.
```
Comment (tarasyyyk) [inline 1-js/03-code-quality/02-coding-style/article.md]:
```suggestion
Більшість лінтерів інтегровані в популярні редактори: просто увімкніть відповідний плаґін в редакторі і налаштуйте стиль.
```

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 3]
```diff
@@ -269,54 +270,55 @@ If you are writing several "helper" functions and the code that uses them, there
...
 
-For instance, for ESLint you should do the following:
+Наприклад, для ESLint вам потрібно зробити наступне:
 
-1. Install [Node.js](https://nodejs.org/).
-2. Install ESLint with the command `npm install -g eslint` (npm is a JavaScript package installer).
-3. Create a config file named `.eslintrc` in the root of your JavaScript project (in the folder that contains all your files).
-4. Install/enable the plugin for your editor that integrates with ESLint. The majority of editors have one.
+1. Встановіть [Node.js](https://nodejs.org/).
+2. Встановіть ESLint використовуючи команду `npm install -g eslint` (npm це менеджер JavaScript пакетів(модулів)).
```
Comment (tarasyyyk) [inline 1-js/03-code-quality/02-coding-style/article.md]:
```suggestion
2. Встановіть ESLint, використовуючи команду `npm install -g eslint` (npm – це менеджер JavaScript пакетів (модулів)).
```

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 4]
```diff
@@ -269,54 +270,55 @@ If you are writing several "helper" functions and the code that uses them, there
...
+3. Створіть файл конфігурації `.eslintrc` у корні вашого JavaScript проекту (у директорії, що містить всі ваші файли).
```
Comment (tarasyyyk) [inline 1-js/03-code-quality/02-coding-style/article.md]:
```suggestion
3. Створіть файл конфігурації `.eslintrc` в корені вашого JavaScript проєкту (у директорії, що містить всі ваші файли).
```

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 5]
```diff
@@ -269,54 +270,55 @@ If you are writing several "helper" functions and the code that uses them, there
...
+4. Встановіть/ввімкніть плагін для вашого редактору, який інтегрується з ESLint. Більшість редакторів мають такий плагін.
```
Comment (tarasyyyk) [inline 1-js/03-code-quality/02-coding-style/article.md]:
```suggestion
4. Встановіть/увімкніть плаґін для вашого редактора, який інтегрується з ESLint. Більшість редакторів мають такий плаґін.
```

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 6]
```diff
@@ -333,16 +335,16 @@ Here's an example of an `.eslintrc` file:
...
 ```
 
-Here the directive `"extends"` denotes that the configuration is based on the "eslint:recommended" set of settings. After that, we specify our own.
+Директива `"extends"` означає, що конфігурація базується на наборі налаштувань "eslint:recommended". Після цього, ви вказуєте ваші власні.
 
-It is also possible to download style rule sets from the web and extend them instead. See <http://eslint.org/docs/user-guide/getting-started> for more details about installation.
+Крім того, можна завантажити набори правил з мережі та розширити їх. Дивіться <http://eslint.org/docs/user-guide/getting-started> для отримання більш детальної інструкції зі встановлення.
 
-Also certain IDEs have built-in linting, which is convenient but not as customizable as ESLint.
+Також, деякі середовища розробки (IDE) мають вбудовані засоби первірки коду, що є зручним, але не таким гнучким в налаштуванні рішенням, як ESLint.
```
Comment (tarasyyyk) [inline 1-js/03-code-quality/02-coding-style/article.md]:
```suggestion
Також, деякі середовища розробки (IDE) мають вбудовані засоби первірки коду, що є зручним, але не таким гнучким в налаштуванні рішенням, як ESLint.
```

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 7]
```diff
@@ -333,16 +335,16 @@ Here's an example of an `.eslintrc` file:
...
+Директива `"extends"` означає, що конфігурація базується на наборі налаштувань "eslint:recommended". Після цього, ви вказуєте ваші власні.
 
-It is also possible to download style rule sets from the web and extend them instead. See <http://eslint.org/docs/user-guide/getting-started> for more details about installation.
+Крім того, можна завантажити набори правил з мережі та розширити їх. Дивіться <http://eslint.org/docs/user-guide/getting-started> для отримання більш детальної інструкції зі встановлення.
 
-Also certain IDEs have built-in linting, which is convenient but not as customizable as ESLint.
+Також, деякі середовища розробки (IDE) мають вбудовані засоби первірки коду, що є зручним, але не таким гнучким в налаштуванні рішенням, як ESLint. 
 
-## Summary
+## Висновки
```
Comment (tarasyyyk) [inline 1-js/03-code-quality/02-coding-style/article.md]:
```suggestion
## Підсумки
```

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
+Насправді, це є мистецтвом програмування -- брати складну задачу і писати код, який одночасно і вирішує задачу, і залишається зрозумілим людині. Саме тут хороший стиль коду стає у нагоді.
 
-## Syntax
+## Синтаксис
 
-Here is a cheat sheet with some suggested rules (see below for more details):
+Деякі запропоновані правила наведені у наступній шпаргалці (дивитись нижче більш детально):
 
 ![](code-style.svg)
 <!--
```
[the remark this answers is not in the document]
Comment (alexgalkin) [reply 1-js/03-code-quality/02-coding-style/article.md]: підправив

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 9]
```diff
@@ -34,15 +34,15 @@ if (n < 0) {
...
 ```
 
-### Curly Braces
+### Фігурні дужки
 
-In most JavaScript projects curly braces are written in "Egyptian" style with the opening brace on the same line as the corresponding keyword -- not on a new line. There should also be a space before the opening bracket, like this:
+У більшості JavaScript проектів фігурні дужки написані у так званому "Єгипетському" стилі з відкриваючою дужкою на тому ж рядку, що й відповідне ключове слово -- не на новому рядку. Також потрібно додавати пробіл перед відкриваючою дужкою, наприклад:
 
 ```js
 if (condition) {
```
[the remark this answers is not in the document]
Comment (alexgalkin) [reply 1-js/03-code-quality/02-coding-style/article.md]: переклав

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 10]
```diff
@@ -34,15 +34,15 @@ if (n < 0) {
...
-Nothing is set in stone here. These are style preferences, not religious dogmas.
+```warn header="Немає обов'язкових правил"
+Залізних правил немає. Все це стильові уподобання, а не релігійні догми.
 ```
 
-### Curly Braces
+### Фігурні дужки
 
-In most JavaScript projects curly braces are written in "Egyptian" style with the opening brace on the same line as the corresponding keyword -- not on a new line. There should also be a space before the opening bracket, like this:
+У більшості JavaScript проектів фігурні дужки написані у так званому "Єгипетському" стилі з відкриваючою дужкою на тому ж рядку, що й відповідне ключове слово -- не на новому рядку. Також потрібно додавати пробіл перед відкриваючою дужкою, наприклад:
```
[the remark this answers is not in the document]
Comment (alexgalkin) [reply 1-js/03-code-quality/02-coding-style/article.md]: виправив

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 11]
```diff
@@ -269,54 +269,55 @@ If you are writing several "helper" functions and the code that uses them, there
...
 
-Linters are tools that can automatically check the style of your code and make improving suggestions.
+Автоматичні засоби перевірки, так звані "лінтери" - це інструменти, що автоматично перевіряють стиль коду та вносять пропозиції щодо його вдосконалення.
 
-The great thing about them is that style-checking can also find some bugs, like typos in variable or function names. Because of this feature, using a linter is recommended even if you don't want to stick to one particular "code style".
+Найкраще в них те, що вони також можуть знайти деякі програмні помилки, наприклад друкарську помилку у назві змінної чи функції. Завдяки цій особливості, рекомендують використовувати лінтер навіть тоді, коли ви не збираєтесь дотримуватись якогось конкретного "стилю коду".
 
-Here are some well-known linting tools:
+Ось декілька 
+Here are some добре відомих засобів для перевірки:
```
Comment (tarasyyyk) [inline 1-js/03-code-quality/02-coding-style/article.md]:
```suggestion
Ось декілька добре відомих засобів для перевірки:
```

Comment (alexgalkin) [reply 1-js/03-code-quality/02-coding-style/article.md]: done

**Pull request merged**